src/wizard/wizard-dialog.ts

```diff
@@ -550,8 +528,12 @@ export class ESPHomeWizardDialog extends LitElement {
       }
     }
 
-    if (changedProps.has("_board") && this._board === "CUSTOM") {
-      this._customBoard = this.shadowRoot!.querySelector("select")!.value;
+    if (changedProps.has("_state") && this._state == "pick_board") {
+      const platform = this._platform.toLowerCase();
+      getSupportedPlatformBoards(platform).then((boards) => {
+        this._supportedBoards = boards;
+        this._busy = false;
```

Comment:
busy is not set to true ? do we also need this? the UI will know it's loading because `this._supportedBoards` is not defined.